pages/[locale]/index.tsx

```diff
@@ -409,7 +415,7 @@ const Index: NextPage<IndexProps> = ({ navItems }: IndexProps) => {
             </Flex.Column>
           ))}
         </ul>
-        <Paragraph size="14px">*{t('index.supportedNetworks.betaWarning')}</Paragraph>
+        <Text.P14 sx={{ pt: '16px' }}>*{t('index.supportedNetworks.betaWarning')}</Text.P14>
```

Comment:
Tiny thing, but I would use margin rather than padding here. You're pushing this whole element down.

Reply:
Got it, thank you! Done! :)